packages/db/schema.ts

```diff
@@ -157,8 +157,11 @@ export const bookmarkAssets = sqliteTable("bookmarkAssets", {
     .primaryKey()
     .$defaultFn(() => createId())
     .references(() => bookmarks.id, { onDelete: "cascade" }),
-  assetType: text("assetType", { enum: ["image"] }).notNull(),
+  assetType: text("assetType", { enum: ["image", "pdf"] }).notNull(),
   assetId: text("assetId").notNull(),
+  content: text("content"),
```

Comment:
let's maybe also store the file name?

Reply:
Well, it's both a negative and a positive thing. 
In case of a "paste" image case, there would be no file name. 
Also many people will upload files with random names. 
I have mixed feelings about this.

Reply:
It's fine for the `paste`s to not have a filename. My plan is to use the filename of assets to be their `title` in the bookmark grid and also give the users the ability to modify it if they want. I checked and this is the default behavior in raindrop for example. Files without a `filename` would just show no title (like they are now). I think the filename in the pdf case would be a nice way to disambiguate the card without having to look into the content itself.